lib/chai/core/assertions.js

```diff
@@ -521,14 +522,26 @@ module.exports = function (chai, _) {
    * @alias greaterThan
    * @param {Number} value
    * @param {String} message _optional_
+   * @namespace BDD
    * @api public
    */
 
   function assertAbove (n, msg) {
     if (msg) flag(this, 'message', msg);
-    var obj = flag(this, 'object');
-    if (flag(this, 'doLength')) {
+    var obj = flag(this, 'object')
+      , doLength = flag(this, 'doLength');
+
+    if (doLength) {
       new Assertion(obj, msg).to.have.property('length');
+    } else {
+      new Assertion(obj, msg).is.a('number');
+    }
+
+    if (_.type(n) !== 'number') {
+      throw new Error('the argument to above must be a number');
```

Comment:
While this isn't the best message, we can leave it for now as it mimicks the others.